setup/features/server-group-launch-settings/aws-ec2/launch-templates.md

```diff
@@ -0,0 +1,322 @@
+Consider trying them out in Deck before using them in API, especially if you are new to them.
+
+## Launch Template Feature Configuration
+Once launch templates are enabled in Clouddriver, a new set of features are unlocked.
+Some of them are tied to the Launch Template directly like `IMDSV2` and 
+some others require a launch template to be used in order to support features like `SpotAllocationStrategy` ([MixedInstancesPolicy](https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_MixedInstancesPolicy.html) features).
+
+
+Review the sections below for details about the features and sample use cases along with Spinnaker API requests. 
+
+### Launch Template Only Parameters
+Review the sections below to determine which features you want to enable in the UI.
+Users will see enabled features as options when configuring a Server Group.
+<table>
+  <thead>
+    <tr>
+      <th>Feature</th>
+      <th style="width=50%;">Description</th>
+      <th>Deck Setting</th>
+      <th>Clouddriver API Request Parameter</th>
+      <th>Release Version</th>
+      <th>Default value in Spinnaker</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>IPv6</td>
+      <td>ASGs associate an IPv6 address to their instances.</td>
+      <td><em>enableIPv6</em></td>
+      <td><em>associateIPv6Address</em></td>
+      <td><em>v1.21</em></td>
+      <td>no default</td>
+    </tr>
+    <tr>
+      <td>IMDSv2</td>
+      <td>Helps mitigate AWS credential theft from the exploitation of SSRF vulnerabilities in web applications. This is only supported by modern SDKs. <font size="-1">Learn more from <a target="_blank" href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/burstable-credits-baseline-concepts.html">AWS</a>.</font></td>
+      <td><em>enableIMDSv2</em></td>
+      <td><em>requireIMDSv2</em></td>
+      <td><em>v1.21</em></td>
+      <td>false</td>
+    </tr>
+    <tr>
+      <td>CreditSpecification</td>
+      <td>The credit option for CPU usage of the instance.
+      Valid for burstable performance EC2 instances only. <font size="-1">Learn more from <a target="_blank" href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/burstable-credits-baseline-concepts.html">AWS</a>.</font></td> 
+      <td><em>enableCpuCredits</em></td>
+      <td><em>unlimitedCpuCredits</em></td>
+      <td><em>v1.24</em></td>
+      <td><a target="_blank" href="https://github.com/spinnaker/clouddriver/blob/master/clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/BasicAmazonDeployHandler.groovy#L488">
+          default <em>false</em></a> used only when all instance types in request support bursting.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Diversified Server Groups with Launch Templates
+Enabling launch templates in Clouddriver also unlocks additional features offered by AWS EC2 AutoScaling that could be helpful for use cases like:
```

Comment:
"Auto Scaling" - two words. Check the correct name for the service. "Amazon EC2 Auto Scaling"?